### ISSUE TITLE
[BUG] Docs website search results don't have scroll.

### ISSUE THREAD

**Issue by akgupta0777** [microsoft/playwright#10167]
**Context:**
- Playwright Version: [what Playwright version do you use?]: Latest
- Operating System: [e.g. Windows, Linux or Mac] : Windows 11
- Node.js version: [e.g. 12.22, 14.6] Latest
- Browser: [e.g. All, Chromium, Firefox, WebKit] Chrome
- Extra: [any specific details about your environment] playwright.dev

**Describe the bug**
Open playwright.dev
search anything in search box.
Observe you cannot scroll down more results after you reach end of screen
Add any other details about the problem here.

https://user-images.githubusercontent.com/51379307/140865991-6ba0eff0-ca72-495a-89bc-0255208002ee.mp4

I think there is no event listeners attached on results of search.

**Comment by aslushnikov** [comment]
@akgupta0777 yeah, the search box is not scrollable by design. If you zoom out the page, you'll see "all search results" button in the end of the list, it'll lead to the following link: https://playwright.dev/search?q=cl

**Comment by akgupta0777** [comment]
Ohh, so in the end you get a link which redirect you to the all search results. This is nice.
But it is a little bit uncomfortable to first zoom out and then see the link. 
A possible solution is you can limit the search results based on screen size and leave the space for the link in the end. This way the user don't have to zoom out it will fit on its screen and then he/she can click on it to see more search results on seperate page.

**Comment by aslushnikov** [comment]
@akgupta0777 This looks like a minor annoyance, so will be pretty low on our list. Feel free to send a PR to https://github.com/microsoft/playwright.dev though!
